README.rst

```diff
@@ -113,10 +119,13 @@ Find label errors with PyTorch, Tensorflow, MXNet, etc. in 1 line of code.
        sorted_index_method='normalized_margin', # Orders label errors
     )
 
-Pre-computed out-of-sample predicted probabilities for CIFAR-10 train set are available here: [`[LINK] <https://github.com/cleanlab/cleanlab/tree/master/examples/cifar10>`__].
-   
-Learning with noisy labels in 3 lines of code!
-==============================================
+**CAUTION:** Predicted probabilities from your model must be out-of-sample! You should never provide predictions on the same datapoints used to train the model, as these will be overfit and unsuitable for finding label-errors. To obtain out-of-sample predicted probabilities for every datapoint in your dataset, you can use `cross-validation <https://machinelearningmastery.com/out-of-fold-predictions-in-machine-learning/>`__. Alternatively it is ok if your model was trained on a separate dataset and you are only evaluating labels in data that was previously held-out.
+
+Pre-computed **out-of-sample** predicted probabilities for CIFAR-10 train set are available: `here <https://github.com/cleanlab/examples/tree/master/cifar10>`__
```

Comment:
Use the sublink, not the main link. e.g. use this direct sublink here: https://github.com/cleanlab/examples/tree/master/cifar10#pre-computed-psx-for-every-noise--sparsity-condition

Also, perhaps this fulfills my comment above, but i still think it would be good somewhere high up in the readme to let people know they can try out cifar-10 with pre-computed pyx. maybe we should just make that a simple tutorial instead though @JohnsonKuan

Reply:
replaced with sublink. added the precomputed statement back in

Reply:
Where do you suggest to move the "Pre-computed **out-of-sample** predicted probabilities for CIFAR-10 train set are available" statement to?

To me it does not make sense earlier on in the readme since the utility of these pyx has not yet been described at an earlier point.